Felienne/hedy
# Fixes highlighting of turtle commands from level 11 onwards

**Description**
The turtle commands were not highlighted as regular commands in level 11 and above. This adds the commands to the list of highlighted commands.

**Fixes #3346 **

**How to test**
Checkout the branch and browse to your local environment. Go to any level 11 - 18, use any of the turtle commands (forward, turn, color) and verify that they are colored like other commands.

**Checklist**
Done? Check if you have it all in place using this list:*
  

If you're unsure about any of these, don't hesitate to ask. We're here to help!
